osi_common.proto

```diff
@@ -154,6 +154,9 @@ message BaseStationary
 
     /// The relative orientation of the landmark w.r.t its parent frame.
     optional Orientation3d orientation = 3;
+    
+    /// The two dimensional (flat) contour of the object. 
```

Comment:
I agree and I think that it is very important that the reasoning you (OSI developers) had in mind when creating OSI, is written down somewhere and explained to all the partners who should be using OSI. We need all the PEGASUS partners on board in order to get through this important initial phase of establishing a common interface.